### ISSUE TITLE
UI/UX: Survey Task's options have low contrast when they have keyboard focus

### ISSUE THREAD

**Issue by shaunanoordin** [zooniverse/Panoptes-Front-End#6089]
## UI/UX and Accessibility Issue

Affects: Survey Tasks

When a Survey Task data option/answer (e.g. "Sleeping", "Playing", etc in Behaviour) has keyboard focus, the label has a low contrast white-text-on-grey-background colour scheme that makes it hard to read.

<img width="360" alt="Screenshot 2022-02-02 at 13 59 37" src="https://user-images.githubusercontent.com/13952701/152177370-2c2b2a81-f5f5-4873-a38f-ea5635ff1147.png">

Line of code responsible is this:

```
.survey-task-choice-answer[data-focused] {
  border: 1px #43bbfd solid;
  color: #fff;
}
```

### Status

Technically low priority, but should be a quick fix. Worth getting done soon since this affects accessibility.
